mllib/src/main/scala/org/apache/spark/ml/clustering/GaussianMixture.scala

```diff
@@ -64,8 +64,8 @@ private[clustering] trait GaussianMixtureParams extends Params with HasMaxIter w
    */
   protected def validateAndTransformSchema(schema: StructType): StructType = {
     SchemaUtils.checkColumnType(schema, $(featuresCol), new VectorUDT)
-    SchemaUtils.appendColumn(schema, $(predictionCol), IntegerType)
-    SchemaUtils.appendColumn(schema, $(probabilityCol), new VectorUDT)
+    val schema_temp = SchemaUtils.appendColumn(schema, $(predictionCol), IntegerType)
```

Comment:
Tiny nit: Java/Scala use camelCase var names. Maybe call this `withPredictionCol` or something, as it's more descriptive anyway.